api/system-certificate/update

```diff
@@ -63,4 +63,20 @@ case $action in
         /sbin/e-smith/signal-event -j certificate-update
        ;;
 
+    "delete")
+
+        file=$(echo $data | jq -r '.file')
+        key=$(echo $data | jq -r '.key')
+        chain=$(echo $data | jq -r '.chain')
+
+        /usr/bin/rm -f "$file"
+        /usr/bin/rm -f "$key"
+        /usr/bin/rm -f "$chain"
+
+        if  [[ $file =~ '/etc/letsencrypt/' ]]; then
+            /usr/sbin/e-smith/config setprop pki LetsEncryptDomains ''
+        fi
```

Comment:
These lines has a serious side effect.

You can have this scenario:
- certificate `le1` with domains `first.domain.org`
- then you add new domain to to `le1` and it becomes `le2`: `first.domain.org, second.domain.org`
- set `le1` as default certificate
- delete `le1` from the web interface
- `le2` will not be renewed anymore

This is not clear even from the doc, but in this case you're not only deleting a certificate, you're also disabling LE at all.

So let's restrict this PR to only deleting the selected certificate.
@filippocarletti do you want to implement another button to totally disable LE?

Reply:
> set le1 as default certificate
> delete le1 from the web interface

the certificate set as default cannot be deleted, does it is a typo @gsanchietti

Reply:
Yes, corrected.

Let's wait thoughts from @filippocarletti

Reply:
this part is still waiting some inputs

Reply:
@gsanchietti is right, we can't set the LetsEncryptDomains pki prop empty, because the renew event exits if it's empty and the cert will not be renewed.

Reply:
Ok, so let's restrict the patch to only removing the cert without editing LE props.

Should the doc changed as well?

Reply:
change done as requested